LayoutTests/platform/glib/TestExpectations

```diff
@@ -469,6 +469,9 @@ http/tests/websocket/tests/hybi/extensions.html [ Pass ]
 
 http/wpt/resource-timing/rt-initiatorType-media.html [ Pass ]
 
+# Damage-related tests
+platform/glib/damage/ [ Pass ]
+
```

Comment:
Now that these are glib specific tests we don't need this, they are expected to pass by default.